web_responsive/readme/DESCRIPTION.rst

```diff
@@ -59,3 +59,7 @@ Features for computers:
 * Sticky chatter topbar
 
   .. image:: ./static/img/chatter_topbar.gif
+
+* AppMenu waits for action finished to show the view
+
+  .. image:: ./static/img/appmenu.gif
```

Comment:
Please put `..` instead of `.` (and you can do the same change in the rest).  With this, you can see from both the generated README and this one properly the images.